R/utils_accessibility.R

```diff
@@ -1,126 +1,129 @@
 #' h1_tabstop
 #'
-#' @param header Heading text
-#' @param tabindex Number for tabindex, default 0
+#' @param heading Heading text
 #' @inheritDotParams shiny::h1
+#' @param tabindex Number for tabindex, default 0
 #'
 #' @return HTML
 #' @export
 #'
 #' @examples
 #' h1_tabstop("Heading")
-h1_tabstop <- function(header, tabindex = 0, ...) {
-  # create header as typical header plus tabindex attribute
-  # ensures header will be stopped at when pressing keyboard tab
-  h1_tabstop <- h1(header, ...) %>%
+h1_tabstop <- function(heading, ..., tabindex = 0) {
```

Comment:
Also, have swapped order of ellipsis (`...`) and `tabindex`. This makes it so `tabindex` does not need to be specified when calling the function, and will correctly use the default value. This same mechanism was already being used in other functions like `nhs_navlistPanel`.